Packs/Druva/Integrations/Druva/Druva.py

```diff
@@ -293,7 +288,7 @@ def main():
             from_date = demisto.args().get('from_date')
             to_date = demisto.args().get('to_date')
             return_outputs(*Druva_QuarantineResource_Command(clientObj, resource_id, resource_type, from_date, to_date))
-            # return_outputs(*Druva_ListQuarantineRanges_Command(clientObj))
+            return_outputs(*Druva_ListQuarantineRanges_Command(clientObj))
```

Comment:
are you sure about the double `return_outputs`?

Reply:
This was done to return the output as requested in the previous review.

Reply:
I meant, if you have the second why do you need the first?

Reply:
the first line indicates success of the operation and second gives more details